spotify/models/playlist.py

```diff
@@ -99,6 +100,7 @@ def __from_raw(self, data: dict) -> None:
         self.images = tuple(Image(**image) for image in data.pop("images", []))
         self.owner = User(client, data=data.pop("owner"))
 
+        self.public = data["public"]
```

Comment:
Style here dictates we exhaust the container, prefer using dict.pop

Reply:
Also please make sure that the `public` field is always present in both simplified and full Playlist objects, If its only partially present under specific conditions you may want to supply a default value such as `None` to indicate the the field was not present.